store/src/main/java/org/apache/rocketmq/store/util/LibC.java

```diff
@@ -50,4 +52,8 @@ public interface LibC extends Library {
     int mlockall(int flags);
 
     int msync(Pointer p, NativeLong length, int flags);
+
+    int mincore(Pointer p, NativeLong length, byte[] vec);
```

Comment:
We do not need to introduce mincore with JNI but use `MappedByteBuffer#isLoaded0` instead. Please see `DefaultMessageStore#checkInMemByCommitOffset` for example.

Reply:
First of all, thank you very much for your review and reply. The MappedByteBuffer#isLoaded0 method was also considered early on to solve this problem, but the granularity of this method is a little somewhat broad (it only returns true if all pages of the detected block of data are in physical memory).

Reply:
isLoaded0

```java
private native boolean isLoaded0(long address, long length, int pageCount);
```

Reply:
In addition, there is already an `isLoaded` method in the MappedFile interface：

```java
/**
 * Check mapped file is loaded to memory with given position and size
 * @param position start offset of data
 * @param size data size
 * @return data is resided in memory or not
 */
boolean isLoaded(long position, int size);
```